connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/CompatibilitySuite.scala

```diff
@@ -152,6 +153,8 @@ class CompatibilitySuite extends ConnectFunSuite {
       ProblemFilters.exclude[Problem]("org.apache.spark.sql.SparkSession.getActiveSession"),
       ProblemFilters.exclude[Problem]("org.apache.spark.sql.SparkSession.clearDefaultSession"),
       ProblemFilters.exclude[Problem]("org.apache.spark.sql.SparkSession.setDefaultSession"),
+      ProblemFilters.exclude[Problem]("org.apache.spark.sql.SparkSession.newSession"),
+      ProblemFilters.exclude[Problem]("org.apache.spark.sql.SparkSession.implicits"),
```

Comment:
Why does this one need to be added? I added `SQLImplicits` yesterday. Can you update your branch?

Reply:
maybe that was forgotten to be added?

Reply:
Ok I removed it..